src/validators/plugins/index.js

```diff
@@ -23,5 +23,5 @@ function validatePlugins(plugins) {
 // plugins can either be a function which has prototype.apply defined,
 // or a function that returns an object with apply property
 export function isPlugin(plugin) {
-  return !!plugin.apply || typeof plugin === 'function'
+  return !!plugin.apply || friendlyTypeOf(plugin) === 'function'
```

Comment:
The difference there is that it's being used to show something to the user. This is the whole reason that `friendlyTypeOf` exists. It's so instead of showing `object` for `/regex/` we show `regex`. However, in this scenario, we're not showing anything to the user at all, we're just verifying that the `plugin` is a function and that's easily done with `typeof` :-)